cldf/sources.bib

```diff
@@ -0,0 +1,9 @@
+@article{Bodt2019b,
+    author = {Aaley, Uday Raj and Bodt, Timotheus A.},
+    date = {2020},
+    day = {08},
```

Comment:
Please don't use date day, month in bibtex, they are barely recognized. Treat this as a normal journal, with volume (3), as this is the third year of the blog, and give it the number of the month, as we have monthly submissions: 

```
volume = {3},
number = {4},
year = {2020},
date = {2020-04-08}, # this is the correct way to display dates
```